### DISCUSSION TITLE
Welcome to rustic Discussions!

### DISCUSSION THREAD

**Discussion by brotzeit** [brotzeit/rustic#489] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pawelkobojek** [comment]
Hi!

I'm a happy user of rustic and just wanted to say thank you to you and all the contributors for your work 👍
